src/program.ts

```diff
@@ -65,6 +72,7 @@ export function createPrograms(input: Array<string>) {
 
     main = path.resolve(main);
     const options = getCompilerOptions(main);
+    options.dtsFiles.forEach(dtsFiles.add, dtsFiles);
```

Comment:
If this is too much "magic" happy to swap it out for a `for...of` loop or a more standard `Array.forEach`.